pgx-pg-sys/build.rs

```diff
@@ -559,8 +559,9 @@ fn build_shim_for_version(
         .unwrap();
     }
 
+    let make = env!("MAKE").unwrap_or("make".to_string());
```

Comment:
In this case `option_env!` is preferable, if `MAKE` is not set with `env!` the build will fail.

Docs: [`env`](https://doc.rust-lang.org/std/macro.env.html) vs [`option_env`](https://doc.rust-lang.org/std/macro.option_env.html)

Reply:
Thank you for the pointer